lib/messageUtils.ts

```diff
@@ -58,12 +60,20 @@ export function getDashboard(
   const encodedBranchSlug = branchSlug ? (encodeURIComponent(branchSlug)) : "";
   const dashboardId = getDashboardIdForTemplate(template);
 
+  // Showing the last 30 complete days to ensure the dashboard isn't including today which has no data yet
+  let encodedSubmissionDate = "30+day+ago+for+30+day";
+  if (startDate && endDate) {
+    encodedSubmissionDate = `${startDate}+to+${endDate}`;
+  } else if (startDate) {
+    encodedSubmissionDate = `${startDate}+to+today`;
+  }
```

Comment:
`branchInfo.nimbusExperiment.endDate` is always null for experiments (I'm assuming because the experiments are still live). In this case, is showing data from `startDate` to today the correct outcome? Or should I show data from `startDate` to the proposed end date, similar to how they're calculated for the date column inside the Skylight tables?

Reply:
I'd say use the proposed end date, except when that is in the past, in which case, just use today.

Reply:
And by "today", I really mean "the most recent data with data", which I guess is generally yesterday.